lgc/patch/llpcSystemValues.cpp

```diff
@@ -62,6 +63,35 @@ void ShaderSystemValues::Initialize(
 
         assert(m_shaderStage != ShaderStageInvalid);
         assert(m_pPipelineState->GetShaderInterfaceData(m_shaderStage)->entryArgIdxs.initialized);
+
+        // Load shadow descriptor table settings from pipeline options.
+        const auto pPipelineOptions = &m_pPipelineState->GetOptions();
+        switch (pPipelineOptions->shadowDescriptorTableConfig)
+        {
+        case 0: // auto; use defaults defined in class
```

Comment:
This is better to define enums rather than use magic numbers
enum
{
    ShadowDescTableAuto = 0,
    ShadowDescTableEnabled = 1,
    ShadowDescTableDisabled = 2,
}

auto -> Auto

Reply:
I have transitioned this to using an enumeration.